src/samples/BotToBot/Bot1/appsettings.json

```diff
@@ -1,29 +1,18 @@
 {
   "TokenValidation": {
     "Audiences": [
-      "00000000-0000-0000-0000-000000000000" // this is the Client ID used for the Azure Bot
-    ]
-  }
+      "{{ClientId}}" // this is the Client ID used for the Azure Bot for Bot1
```

Comment:
Don't we need to add Bot2 to make sure we validate the incoming token (from bot2) with the appropriate claim `aud` ?